packages/oasis3-mct/package.py

```diff
@@ -215,7 +222,10 @@ def edit(self, spec, prefix):
         config = {}
 
         # TODO: https://github.com/ACCESS-NRI/ACCESS-OM/issues/12
-        NCI_OPTIM_FLAGS = "-g3 -O2 -axCORE-AVX2 -debug all -check none -traceback"
+        if "@access-esm1.5" in self.spec:
+            NCI_OPTIM_FLAGS = "-g3 -O2 -march=cascadelake -axCORE-AVX512 -debug all -check none -traceback"
+        else:
+            NCI_OPTIM_FLAGS = "-g3 -O2 -axCORE-AVX2 -debug all -check none -traceback"
```

Comment:
Set the default `NCI_OPTIM_FLAGS`, as before. Then for each special case, override the variable.

Reply:
OK in general. In this instance there is only one special case, so leaving as-is for now.

Reply:
No, there are three:
```
        if "@access-esm1.5" in self.spec:
            NCI_OPTIM_FLAGS = "-g3 -O2 -xCORE-AVX512 -debug all -check none -traceback"
        else:
            NCI_OPTIM_FLAGS = "-g3 -O2 -axCORE-AVX2 -debug all -check none -traceback"
        CFLAGS = ""
        if "+deterministic" in self.spec:
            NCI_OPTIM_FLAGS = "-g0 -O0 -axCORE-AVX2 -debug none -check none"
            CFLAGS = "-g0"

        if "+optimisation_report" in self.spec:
            NCI_OPTIM_FLAGS += " -qopt-report=5 -qopt-report-annotate"
```

Reply:
Addressed. Please check.